acceptance/helpers/beaker_helper.rb

```diff
@@ -280,6 +349,37 @@ def hosts_without_role(hosts, role_to_exclude)
       host unless host["roles"].include?(role_to_exclude.to_s)
     end
   end
+
+  # Runs the command on the specified host
+  #
+  # @param host The unix style host
+  # @param command The command to run
+  #
+  # @example
+  #   output = run_command(host, command)
+  #
+  def run_beaker_command(host, command)
+    puts "command: #{command}"
+    output = on(host, command).stdout.rstrip
+    puts "output:"
+    puts output
+    return output
+  end
+
+  # Runs the mock puppet agent on the specified host
+  #
+  # @param host The unix style host
+  #
+  # @example
+  #   output = run_mock_puppet_agent(host)
+  #
+  def run_mock_puppet_agent(host)
+    fake_puppet_path = "/tmp/ref_arch_setup/puppet-enterprise-*"
+    command = "#{fake_puppet_path}/puppet agent -t"
+
+    output = run_beaker_command(host, command)
```

Comment:
Run_beaker_command only gets used once.  And the main thing it seems to add is printing the output.  Which I thought already happened with "on" commands.  And if it doesn't, maybe it has a setting to make it do so.  If not, add more comment on run_beaker_command as to how it adds value over the on commands.

Reply:
Removed

Reply:
Removed